ide-support/revliburl.livecodescript

```diff
@@ -3945,6 +3946,9 @@ on ulSendCallback pUrl, pStatus
   if exists(tObject) then
     ## need quotes for the url formatting and the possibility of multiple items in second argument
     send tMessage &&  quote & pUrl & quote & comma & quote & pStatus & quote to tObject in 0 milliseconds
+   else
+    # If no target then just send to self so it propagates through message path.
+    send tMessage &&  quote & pURL & quote & comma & quote & pStatus & quote to me in 0 milliseconds
```

Comment:
@trevordevore It seems like using the 
```
send tMessage && "pUrl, pStatus"
```
form would make sense here too. And perhaps the string could be constructed outside of the if loop to save having it in two places?